cargo-dist/src/lib.rs

```diff
@@ -433,7 +433,7 @@ fn generate_unified_checksum(
                 continue;
             }
 
-            writeln!(&mut output, "{checksum} {artifact_name}").unwrap();
+            writeln!(&mut output, "{checksum}  {artifact_name}").unwrap();
```

Comment:
it would be useful to put a comment here explaining that there are purposefully 2 spaces here and why- since it's hard to know from just this line :)

Reply:
Good call!